PC/config_minimal.c

```diff
@@ -5,16 +5,20 @@
 
 #include "Python.h"
 
+#ifdef Py_ENABLE_SHARED
 /* Define extern variables omitted from minimal builds */
 void *PyWin_DLLhModule = NULL;
+#endif
 
 
 extern PyObject* PyInit_faulthandler(void);
 extern PyObject* PyInit__tracemalloc(void);
 extern PyObject* PyInit_gc(void);
 extern PyObject* PyInit_nt(void);
 extern PyObject* PyInit__signal(void);
+#if defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_SYSTEM) || defined(MS_WINDOWS_GAMES)
```

Comment:
@zooba, since the check for building winreg and mmap occurs in several places (here, in "PC/config.c", and the respective modules), maybe `HAVE_WINREG_MODULE` and `HAVE_MMAP_MODULE` should be defined in "PC/pyconfig.h".